packages/core/src/billings.ts

```diff
@@ -66,8 +65,8 @@ export class Billings {
     year: string;
     month: string;
   }): Promise<Invoice> {
-    const { orgId } = request;
-    const queryString: string = makeQueryString({});
+    // const { orgId } = request;
+    // const queryString: string = makeQueryString({});
```

Comment:
I commented out these lines becuase the function has "todo: implement" comment.
I thought that those lines will be used later when we implement the function.